src/components/Card/CardFooterCTA.js

```diff
@@ -3,13 +3,14 @@ import PropTypes from 'prop-types'
 import classNames from 'classnames'
 
 const propTypes = {
-  className: PropTypes.string
+  className: PropTypes.string,
+  children: PropTypes.node
 }
 
 const CardFooterCTA = props => {
-  const { className, ...attributes } = props
+  const { className, children } = props
   const classes = classNames(className, 'it-card-footer')
-  return <div className={classes} {...attributes} />
+  return <div className={classes}>{children}</div>
```

Comment:
Can you restore it with the old version?

The problem with the new one is that you are only letting the `children` props pass. Say the developer wants to add an `aria-something` on the `div`, how can he does that?

`attributes` contains `children` and also many HTML attributes in it.